AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6511 @@
+{
```

Comment:
<p>nit: change x axis to Prior belief of conversion rate to make this unambiguous</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Done</p>